fdb-relational-api/src/main/java/com/apple/foundationdb/relational/api/Options.java

```diff
@@ -549,6 +555,7 @@ private static Map<Name, List<OptionContract>> makeContracts() {
         data.put(Name.ENCRYPT_WHEN_SERIALIZING, List.of(TypeContract.booleanType()));
         data.put(Name.ENCRYPTION_KEY_STORE, List.of(TypeContract.nullableStringType()));
         data.put(Name.ENCRYPTION_KEY_ENTRY, List.of(TypeContract.nullableStringType()));
+        data.put(Name.ENCRYPTION_KEY_ENTRY_LIST, List.of(new CollectionContract<>(TypeContract.stringType())));
```

Comment:
Does this contract cover ordering? Do we care about that from a TypeContract perspective?

Reply:
I guess it's better to be explicit about requiring that the collection be ordered.